lib/features/children/generosity_challenge/cubit/generosity_challenge_cubit.dart

```diff
@@ -166,7 +166,7 @@ class GenerosityChallengeCubit extends Cubit<GenerosityChallengeState> {
         final timeDifference =
             state.unlockDayTimeDifference == UnlockDayTimeDifference.days
                 ? diff.inDays
-                : diff.inMinutes;
+                : diff.inSeconds;
```

Comment:
Maybe this needs to be reverted? :)

Reply:
Probs easier to revert, my changes will switch immediately anyways, not after a minute.